### PR TITLE
More readable appveyor.yml

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+  - PYTHON: 'C:\Python34-x64'
+  - PYTHON: 'C:\Python35'
+  - PYTHON: 'C:\Python35-x64'
 matrix:
   fast_finish: true
 init:
-- "ECHO %PATH%"
-- "ECHO %PYTHON%"
-- ps: "ls C:/Python*"
+- ps: ls C:/Python*
```
Comment (QuLogic) [inline appveyor.yml]: Why do you even need this one, anyway?

Comment (asottile) [inline appveyor.yml]: we don't _need_ it, but it's nice to know what appveyor supports out of the box (was certainly useful when we were considering adding python3.5)

Comment (QuLogic) [inline appveyor.yml]: They've got [documentation for that](https://www.appveyor.com/docs/installed-software#python), though.

Comment (asottile) [inline appveyor.yml]: Indeed.  However the docs were out of date when python3.5 was added.

**Pull request merged**